app/react/portainer/environments/update-schedules/common/ScheduledTimeField.tsx

```diff
@@ -31,7 +31,15 @@ export function ScheduledTimeField({ disabled }: Props) {
         <DateTimePicker
           format="y-MM-dd HH:mm:ss"
           className="form-control [&>div]:border-0"
-          onChange={(date) => setValue(isoDate(date.valueOf()))}
+          onChange={(date) => {
+            let dateToSave = date;
+            // handle clear button
+            if (!date) {
+              dateToSave = new Date(Date.now() + 24 * 60 * 60 * 1000);
+            }
```

Comment:
can use const: `const dateToSave = date || new Date(Date.now() + 24 * 60 * 60 * 1000)`

or can give a default value to the date parameter, but that may not work if `DateTimePicker` sends `null`

Reply:
DateTimePicker sends null when you clear the date using the X